global.json

```diff
@@ -1,3 +1,6 @@
 {
-    "projects": ["src"]
+    "projects": ["src"],
+    "sdk": {
+       "version": "1.0.0-preview2-003154"
```

Comment:
I just kept it consistent within the repo. Out repos are inconsistent with spacing in global.json and since it's not there anymore in dev, I don't care enough to update them in the patch branches.